### ISSUE TITLE
Jailbreak Trail showing when dead

### ISSUE THREAD

**Issue by Shadow132** [adamdburton/pointshop#192]
When I join my serve and wait to respawn everyone sees my trails and it never goes way

**Comment by HandsomeMatt** [comment]
yeah we had an issue report on this literally this morning, I'll merge in
the fix when I get home

**Comment by Shadow132** [comment]
ok thank you

**Comment by HandsomeMatt** [comment]
Actually it was you yesterday.. Did the isSpectating check not work?

**Comment by Shadow132** [comment]
yea

**Comment by Shadow132** [comment]
I was working on it then i put in isSpectating and it didnt work but I though it did because i had holster my trail thinking i did fix it

**Comment by Shadow132** [comment]
Is there any possible way of fixing this?

**Comment by Shadow132** [comment]
Could you help me out cause this issue is really annoying!
